app/services/course/assessment/submission/update_service.rb

```diff
@@ -138,22 +130,25 @@ def create_missing_submission_questions
     new_submission_questions = []
     questions_without_submission_questions.each do |question|
       new_submission_questions <<
-        @submission.submission_questions.build(question: question)
+        Course::Assessment::SubmissionQuestion.new(submission: @submission, question: question)
```

Comment:
Instead of using build, we use new to create new records here to allow the submission_question acting_as (discussion post) to be bulk inserted using activerecord-import